### ISSUE TITLE
Trying to build something makes me crash

### ISSUE THREAD

**Issue by Hoodiebud** [OpenRCT2/OpenRCT2#5948]
Running openrct2 launcher version v0.1.1-developbuild fab8efb. Im on windows 10
![skarmbild 50](https://user-images.githubusercontent.com/30286687/28365342-7d82f810-6c88-11e7-894d-c1731213b6e0.png)
This is what happens when i try to build something. I got this both on multiplayer and singleplayer. Help plz.

**Comment by Broxzier** [comment]
This has already been fixed with #5947.

**Comment by Gymnasiast** [comment]
Indeed, this has been fixed today. Update to the newest version and you should be good.
